doc/1/plugins/guides/events/core-hotelclerk-addSubscription/index.md

```diff
@@ -0,0 +1,29 @@
+---
+code: true
+type: page
+title: core:hotelClerk:addSubscription
+---
+
+# core:hotelClerk:addSubscription
+
+
+| Arguments  | Type              | Description                           |
+| ---------- | ----------------- | ------------------------------------- |
+| `diff`     | <pre>object</pre> | Contains information for subscription |
+
+Triggered whenever a [subscription](/core/1/api/controllers/realtime/subscribe) is added.
+
+## diff
+
+The provided `diff` object has the following properties:
+
+| Properties     | Type                 | Description                                                                                                       |
+| -------------- | -------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `roomId`       | <pre>integer</pre>   | The new room unique identifier                                                                                    |
+| `connectionId` | <pre>integer</pre>   | The [ClientConnection](/core/1/protocols/api/context/clientconnection) unique identifier                          |
+| `index`        | <pre>string</pre>    | Index                                                                                                             |
+| `collection`   | <pre>string</pre>    | Collection                                                                                                        |
+| `filters`      | <pre>object</pre>    | [Filters](https://docs.kuzzle.io/core/1/guides/cookbooks/realtime-api/terms/) to match with                       |
+| `changed`      | <pre>boolean</pre>   | Indicates if a channel was associated to the current `roomId` or if the room was associated to the `connectionId` |
```

Comment:
So, `true` mean what ?

Reply:
Well, it means "a channel was associated to the current `roomId` or the room was associated to the `connectionId`" :)

Reply:
Your explanation is hard to get because it's not written correctly.

Since I do know what this flag does, here is a proposal:

```
| `changed`      | `true` or `undefined`   | If present, indicates that a new client or a new channel has been created (triggers a cluster sync) |
```

But because the information provided by this flag is only for internal purposes (it tells that internal structures have changed and should be resync across the cluster), I suggest to leave it undocumented.